package.json

```diff
@@ -7,7 +7,7 @@
   "types": "target/index.d.ts",
   "scripts": {
     "lint": "tsc --noEmit && yarn lint-es",
-    "lint-es": "eslint --ext .ts,.tsx",
+    "lint-es": "eslint . --ext .ts,.tsx",
```

Comment:
I cannot add a suggestion because there are deleted lines but maybe we could use the `pre` suffix here instead?

```json
    "prelint": "tsc --noEmit ",
    "lint": "eslint --ext .ts,.tsx",
```

Same could happen with the `test` script:

```json
    "pretest": "yarn lint",
    "test": "jest",
```

Obviusly this is just a minor cosmetic change, I can totally live without it :smile: